### PR TITLE
add angular-bootstrap-colorpicker v3.0.18 with git autoupdate, close …

### PR DESCRIPTION
**Pull request by maruilian11**
…#4975
@Piicksarn could u plz help me check this PR for #4975, thank you!
**git repo url:** https://github.com/buberdds/angular-bootstrap-colorpicker
**Watch 15
  Star 248
 Fork 150**
* [x] Not found on cdnjs repo
* [x] No already exist issue and PR
* [x] Notable popularity
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)
* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by maruilian11** [comment]
@Amomo could i help me check this PR? thank you very much!

File: ajax/libs/angular-bootstrap-colorpicker/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "author": "Michal Zielenkiewicz",
+  "license": "MIT",
+  "homepage": "https://github.com/buberdds/angular-bootstrap-colorpicker",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/buberdds/angular-bootstrap-colorpicker.git",
+    "basePath": "",
+    "files": [
+      "js/*.js",
+      "img/*.img",
```
Comment (Amomo) [inline ajax/libs/angular-bootstrap-colorpicker/package.json]: There are `png` files in the img folder~

**Comment by Amomo** [comment]
@maruilian11 Ummm...because they are not under a dist/ folder, I am not sure for using wildcard like this.
@PeterDaveHello Is this case ok?

**Comment by PeterDaveHello** [comment]
Of course not ok...

**Comment by Amomo** [comment]
Oh! Thanks!
@maruilian11 please point to existing files in a non-dist folder, don't add every files by wildcard.

**Comment by maruilian11** [comment]
@Amomo is that ok?

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you~ :-D

**Comment by PeterDaveHello** [comment]
I didn't see the images under `img` older been referenced by the css or js file, do you really need them?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, i will check it next time!
@Amomo could u help me check it again? thank you!

**Comment by Amomo** [comment]
@maruilian11 I found it has a new version v3.0.19 now. 😱
Could you modify your PR? Thank you!

**Comment by maruilian11** [comment]
@Amomo but i add auto-update:joy: can we let it go:joy::joy::joy:

**Comment by Amomo** [comment]
@PeterDaveHello there are no different in js and css files between v3.0.18 and v3.0.19.
And maruilian11 has add auto-update config in package.json.
I think it is fine to use PR, could you take a look? Thank you!

**Comment by PeterDaveHello** [comment]
If so, the auto-update mechanism and config will not be useful in this case, what's the reason why you think we can still do it the same way? Even v3.0.17 has the same files, are you sure this is good?

**Comment by maruilian11** [comment]
@Amomo i have add the latest version, could u help me check it? thank you!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you!

**Pull request merged**